### PR TITLE
Add M1 Compatibility & Pull in 2022 ESK updates

### PR DESCRIPTION
**Pull request by bchiang7**
## Context 

Was trying to run the ups.com repo (which is based on Skela) and hit some snags going through the setup steps with my M1 mac!

## Changes

Updates from:
https://github.com/Upstatement/editorial-starter-wp-theme/pull/179/files
https://github.com/Upstatement/editorial-starter-wp-theme/pull/190
https://github.com/Upstatement/editorial-starter-wp-theme/pull/191
and others

## How To Test

With an M1 mac go through installation steps https://github.com/Upstatement/skela-wp-theme#-installation

- `./bin/uninstall` (if you've installed skela before)
- `nvm install`
- Duplicate the contents of `.env.sample` into a new `.env` file if you don't already have it
- `./bin/install`
- Confirm installation is successful then `./bin/start` and confirm the following works
  - http://skela.ups.dock and http://skela.ups.dock/wp-admin (`admin`/`password`)
  - http://localhost:3001 and http://localhost:3001/wp-admin (`admin`/`password`)
  - And skela shows up in your ups dock dashboard http://ups.dock/
    image

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -11,5 +11,5 @@ parameters:
         - vendor/php-stubs/acf-pro-stubs/acf-pro-stubs.php
     paths:
         - src/
-    excludes_analyse:
+    excludePaths:
```
Comment (bchiang7) [inline phpstan.neon.dist:14]: `excludes_analyse` is deprecated

File: phpcs.xml.dist [hunk 2]
```diff
@@ -2,6 +2,12 @@
 <ruleset name="Ups">
   <description>Baseline coding standards</description>
 
+  <!--
+  Prevent errors caused by WordPress Coding Standards not supporting PHP 8.0+.
+  See https://github.com/WordPress/WordPress-Coding-Standards/issues/2035
+  -->
+  <ini name="error_reporting" value="E_ALL &#38; ~E_DEPRECATED" />
```
Comment (bchiang7) [inline phpcs.xml.dist:9]: Previously, we just told developers to use a PHP version < 8.0 (e.g. https://github.com/Upstatement/pratt-wp-theme#-system-requirements). This suppresses WPCS errors for PHP 8.0+ so we don't have to remember to do that

**Review by mgburns — APPROVED** [review]
@bchiang7 Thanks for pulling these fixes in! I didn't have time to actually set up the environment (would appreciate an assist from @braican on that!) but I did review the changes and they all LGTM.

**Comment by bchiang7** [comment]
@ALJ @braican @gvjacob would really appreciate a quick review this week! these are all changes that have been implemented in WSK (mainly authored by gino), so I'm not so worried about bugs. I just want to make sure it doesn't just work on my machine before I merge

**Comment by gvjacob** [comment]
@bchiang7 Install went through! I was able to start the app but it's https://skela.ups.dock is showing the Ups Dock dashboard only without the Skela app there.

CleanShot 2023-01-18 at 10 09 44

**Comment by bchiang7** [comment]
@gvjacob hmm I just tried again after restarting ups dock because it seemed like my ups.dock dashboard was cached and it still works for me. could you test again with these extra steps?

- `./bin/uninstall`
- cd to your `ups-dock` directory in your terminal
- run `docker-compose down`
- then run `docker-compose up -d`
- cd back to the skela directory then run through `./bin/install` steps again

Looks like this for me
image

**Comment by gvjacob** [comment]
@bchiang7 hmm the URL is still resolving to the ups dock dashboard for me after those steps. I'll give it a look a bit later today but would also appreciate a look from @ALJ and @braican

**Review by ALJ — APPROVED** [review]
Hey @bchiang7 this PR worked exactly as advertised on my MacBook Pro M1 🎉  I don't know if this has anything to do with @gvjacob 's issue: I had _never_ run Skela before; this PR was my first attempt

**Comment by bchiang7** [comment]
> @bchiang7 hmm the URL is still resolving to the ups dock dashboard for me after those steps. I'll give it a look a bit later today but would also appreciate a look from @ALJ and @braican

@gvjacob Hmmmmm I just compared all the docker related files to WSK and they're identical, save for the few places I replaced `editorial-starter.ups.dock` with `skela.ups.dock`. Did you remember to copy over the `.env` file? I wonder if installing WSK works on your machine?

**Review by braican — APPROVED** [review]
Works as expected on my end.

**Pull request merged**